source/_integrations/frankever.markdown

```diff
@@ -0,0 +1,49 @@
+---
+title: FrankEver
+description: Integrate FrankEver (Powered by Shelly) devices
+ha_category:
+  - Binary sensor
+  - Climate
+  - Cover
+  - Energy
+  - Event
+  - Light
+  - Number
+  - Select
+  - Sensor
+  - Switch
+  - Text
+  - Update
+  - Valve
+ha_release: 2025.4
+ha_codeowners:
+  - '@balloob'
+  - '@bieniu'
+  - '@thecode'
+  - '@chemelli74'
+  - '@bdraco'
```

Comment:
I copied the code owners list from Shelly, please let me know if you don't want to be listed here and I'll update the PR.

Reply:
This will be automatically synced with the parent integration.